### PR TITLE
Expose the skipPointerEventsCheck check

### PR DESCRIPTION
**Pull request by joebingham-wk**
## Motivation
  

Since we've last generated our js src and wrapped the APIs, the parameter `skipPointerEventsCheck` has been exposed on UserEvent clicking APIs. Using it isn't a super rare use case, so this PR exposes the parameter to give consumers access.

## Changes
  
- Consume the last minor version of user-event v13 (v14 was just released two weeks ago)
- Add the skipPointerEventsCheck option to `click` and `dblClick`
- Add tests

## Review
_[See CONTRIBUTING.md][contributing-review-types] for more details on review types (+1 / QA +1 / +10) and code review process._

  

### QA Checklist
- [ ] Tests were updated and provide good coverage of the changeset and other affected code
- [ ] Manual testing was performed if needed
    - [ ] Steps from PR author:
        
      - [ ] CI passes and tests correctly exercise the parameter
    - [ ] Anything falling under manual testing criteria [outlined in CONTRIBUTING.md][contributing-manual-testing]

## Merge Checklist
While we perform many automated checks before auto-merging, some manual checks are needed:
- [ ] A Client Platform member has reviewed these changes
- [ ] There are no unaddressed comments _- this check can be automated if reviewers use the "Request Changes" feature_
- [ ] _For release PRs -_ Version metadata in Rosie comment is correct

[contributing-review-types]: https://github.com/Workiva/react_testing_library/blob/master/CONTRIBUTING.md#review-types
[contributing-manual-testing]: https://github.com/Workiva/react_testing_library/blob/master/CONTRIBUTING.md#manual-testing-criteria

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Comment by joebingham-wk** [comment]
The dev build is failing because pub is deprecated and is getting removed

File: lib/src/user_event/user_event.dart [hunk 1]
```diff
@@ -63,6 +63,13 @@ abstract class UserEvent {
   /// [UIEvent.detail](https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/detail)
   /// for more information.
   ///
+  /// ### [skipPointerEventsCheck]
```
Comment (sydneyjodon-wk) [inline lib/src/user_event/user_event.dart:66]: It looks like this option is also available on `hover`, `unhover`, `selectOptions`, and `deselectOptions` - do we want to add it to those methods in this PR as well?

File: test/unit/user_event/click_test.dart [hunk 2]
```diff
@@ -97,6 +97,27 @@ void main() {
         _verifyClickEvent(clickCount: clickCount);
       });
 
+      test('skipPointerEventsCheck', () {
+        var wasClicked = false;
+
+        // Render into a local view because there isn't a great
+        // way to test pointer events with shared test logic
+        final localView = rtl.render(react.button({
+          'data-test-id': 'the-local-button',
```
Comment (sydneyjodon-wk) [inline test/unit/user_event/click_test.dart:106]: #nit you could probably avoid having to make a separate view and getting by test-id if you use `view.rerender` instead.

File: lib/src/user_event/user_event.dart [hunk 3]
```diff
@@ -63,6 +63,13 @@ abstract class UserEvent {
...
+  ///
+  /// Use [skipPointerEventsCheck] to skip checking if any element
+  /// in the DOM-tree has `'pointer-events: none'` set. This check is
```
Comment (sydneyjodon-wk) [inline lib/src/user_event/user_event.dart:69]: Maybe could also add that `click` will usually throw an error if `'pointer-events: none'` is set - so this is changing that

**Review by sydneyjodon-wk — CHANGES_REQUESTED** [review]
This option will be really nice!

**Comment by joebingham-wk** [comment]
Good catches, @sydneyjodon-wk! All feedback addressed!

**Review by sydneyjodon-wk — APPROVED** [review]
+10

**Review by greglittlefield-wf — APPROVED** [review]
+1

@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**